api/src/feature_queries.py

```diff
@@ -1,5 +1,36 @@
 from src.embedder_engine import embedder
 
+def keyword_query_v3(phrase):
+    """
+    """
+    q = {
+        "query": {
+            "bool": {
+                "should": [
+                    {
+                        "match_phrase": {
+                            "name": phrase
```

Comment:
New keyword query shared on slack. Without fuzzy matching. Used together with our semantic search implementation for the latest hybrid search.